### PR TITLE
Add vakar bot skeleton

### PR DESCRIPTION
**Pull request by vakar8888**
Howto run (first run):

- cd to dir bot
- npm install
- run ./testnetify.sh to connect to testnet
- run node toto.js
- enter name (can be changed later in ~/.config/lotto-bot/conf.js)
- enter passphrase (just press enter, passphrase is in code)
- on screen you should see the pairing code for your wallet (bot)
- use a GUI wallet (not testnet wallet!) to connect to the bot  and try the commands
- to use normal GUI wallet for testnet, install it, run ./testnetify.sh and change hub to testnet hub
- in bot/conf.js set constants WINSTON_LOG_DIR and control_addresses

### CODE REVIEW

File: bot/toto.js [hunk 1]
```diff
@@ -0,0 +1,790 @@
...
+	  })
+	]
+});
+
+// add more logger if needed
+//...
+
+/********************************************************/
+// headless code
+/********************************************************/
```
Comment (junakowicz) [inline bot/toto.js:62]: Can we split out `headless code` to separate file and require it, should be easier to manage.

Comment (junakowicz) [reply bot/toto.js:62]: ideally this file would be our main bot only file, rest of helpers/libraries could be required/ pulled in. this should help in testing and dividing work

**Comment by thedavidmeister** [comment]
@vakar8888 i will work on getting all these steps built into the provisioning process

**Comment by vakar8888** [comment]
> enter passphrase
First, we should decide how this works. You have another solution already built in.

> use a GUI wallet
GUI wallet should not be part oft the build. Each of us has own Testnet Wallet(s) and they should be on e.g. on the host. You can reuse them all the time. If you build them, you get each time new keys and new addresses, except when you include the key-file (keys.json) in the build. But then we have the same wallet running more than once. This will cause issues.

**Review by junakowicz — COMMENTED** [review]
general comments as @thedavidmeister said I think if we are moving to bot dir then old files should be removed or better moved to /bot in first commit then modified

**Comment by vakar8888** [comment]
> Can we split out headless code to separate file and require it, should be easier to manage.
> ideally this file would be our main bot only file, rest of helpers/libraries could be required/ pulled in. this  should help in testing and dividing work

sure, I will split it

**Review by junakowicz — APPROVED** [review]

**Pull request merged**